<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CS:GO Bomb Timer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 10, 10);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
	border: 2px solid;
	border-color: rgb(20, 20,20);
	color: rgb(255, 255, 255);
	width: 20px;
	font:  bold 20px;
}

QProgressBar::chunk{
    background-color: rgb(30, 30, 30);
	width: 1px;
}</string>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>PRESS START TO LISTEN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="output_field">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser{
color: rgb(255, 255, 255);
border: 1px solid;
border-color: rgb(30, 30, 30);
}</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="start_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(50, 50, 50);
border-style: outset;
font: bold 8pt;
padding: 6px;
color: white;
}

QPushButton:hover:!pressed
{
  background-color: rgb(70, 70, 70);
}

QPushButton:disabled
{
	background-color: rgb(20, 20, 20);
	color: rgb(100,100,100);
}
</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/Icons/start_icon_inactive.png</disabledoff>
          <activeoff>:/Icons/start_icon.png</activeoff>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(50, 50, 50);
border-style: outset;
font: bold 8pt;
padding: 6px;
color: white;
}

QPushButton:hover:!pressed
{
  background-color: rgb(70, 70, 70);
}

QPushButton:disabled
{
	background-color: rgb(20, 20, 20);
	color: rgb(100,100,100);
}
</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/Icons/stopLog_icon.png</normaloff>
          <disabledoff>:/Icons/stopLog_icon_inactive.png</disabledoff>
          <activeoff>:/Icons/stopLog_icon.png</activeoff>:/Icons/stopLog_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settings_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(50, 50, 50);
border-style: outset;
font: bold 8pt;
padding: 6px;
color: white;
}

QPushButton:hover:!pressed
{
  background-color: rgb(70, 70, 70);
}

QPushButton:disabled
{
	background-color: rgb(20, 20, 20);
	color: rgb(100,100,100);
}
</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/Icons/options_icon.png</normaloff>
          <disabledoff>:/Icons/options_icon_inactive.png</disabledoff>
          <activeoff>:/Icons/options_icon.png</activeoff>:/Icons/options_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
